### PR TITLE
Add run-python-script

### PR DESCRIPTION
**Pull request by zakhark**
#42

### CODE REVIEW

File: checks/load.go [hunk 1]
```diff
@@ -21,6 +21,10 @@ var availableChecks = map[string]ArgableFunc{
 		args["interpreter"] = "/bin/bash"
 		return RunScript{}.FromArgs(args)
 	},
+	"run-python-script": func(args map[string]interface{}) (Checker, error) {
+		args["interpreter"] = "python"
```
Comment (chasinglogic) [inline checks/load.go]: Will this not need to be a full path? (i.e. do your tests pass I guess?)

File: checks/load.go [hunk 2]
```diff
@@ -21,6 +21,10 @@ var availableChecks = map[string]ArgableFunc{
...
+		return RunScript{}.FromArgs(args)
+	},
```
Comment (chasinglogic) [inline checks/load.go]: Can we add specific ones for like `run-python2-script` and `run-python3-script`?

File: checks/load.go (hunk 1 above)
Comment (chasinglogic) [reply checks/load.go]: This works I’m wrong.

File: checks/load.go (hunk 2 above)
Comment (zakhark) [reply checks/load.go]: Added

**Pull request merged**